01week/datatypes.js

```diff
@@ -0,0 +1,76 @@
+ 'use strict';
+
+<!-- #1 - Display the Date -->
+
+function ReturnDate() {
+const date = new Date;
```

Comment:
Also, please go to this branch in your terminal and run `npm run lint`. You need to indent your functions.